src/core/dxf/qgsdxfexport.h

```diff
@@ -95,9 +98,33 @@ class CORE_EXPORT QgsDxfExport : public QgsLabelSink
          */
         QString splitLayerAttribute() const;
 
+        /**
+         * \brief Flag if data defined point block symbols should be created. Default is false
+         * \return True if data defined point block symbols should be created
+         * \since QGIS 3.36
+         */
+        bool buildDDBlocks() const { return mBuildDDBlocks; }
+
+        /**
+         * \brief
+         * \return
+         * \since QGIS 3.36
+         */
+        int DDBlocksMaxNumerOfClasses() const { return mDDBlocksMaxNumberOfClasses; }
```

Comment:
```suggestion
        int dataDefinedBlocksMaximumNumberOfClasses() const { return mDDBlocksMaxNumberOfClasses; }
```

or something more concise without abbreviations :laughing: